_posts/2018-07-31-how-to-move-to-https.md

```diff
@@ -0,0 +1,100 @@
+<br> ![Chrome misconfigured HTTPS]({{ site.url }}/assets/img/blog/misconfigured-https.png)
+
+### The good news about moving to HTTPS
+
+Ensuring your website serves assets exclusively over HTTPS is not as hard as it used to be, and is getting easier by the day.
+
+#### There are free digital certificates
+
+There's no such thing as a free lunch, but a free certificate from a reputable CA? It would seem so. People are just giving them away these days… Seriously, many years in the making since its founding by two Mozilla employees in 2012, the Let's Encrypt project has vowed to make the web a secure space and has successfully endeavored to become a trusted CA that literally does not charge for the certificates they provide. They have shorter lease cycles of 60 to 90 days, but they also offer tooling around automating the process of reinstalling newly provided certificates.
+
+#### There are easier and cheaper ways to install certificates
+
+With the advent of the aforementioned free certificate, many platform-as-a-service (PaaS) hosting options have incorporated low cost or free installation of certificates through their hosting platform sometimes as easily as a few clicks. Let's Encrypt has been [ adopted across a broad range of website hosting providers](https://community.letsencrypt.org/t/web-hosting-who-support-lets-encrypt/6920) like Squarespace, GitHub Pages, Dreamhost all of which we use alongside many others.
+
+For many of our Drupal partners, we prefer to use a platform as a service (PaaS) hosting option like Pantheon, Acquia, or Platform.sh. Both [Pantheon](https://pantheon.io/features/managed-https
+) and [Platform.sh](https://platform.sh/blog/free-ssl-certificates-for-every-project-every-environment) now provide a free HTTPS upgrade for all hosting plans; Acquia Cloud, another popular Drupal PaaS, is [still a bit behind in this regard](https://redfinsolutions.com/blog/installing-free-lets-encrypt-ssl-certificates-acquia#comment-3290247800). We have found that the efficiency gains of spending less time in server administration translates to more value to our clients empowering additional effort for the strategy, design, and development for which they hired us. In addition to efficiency, the reliability and consistency provided by finely tuned PaaS offerings are, in most cases, superior to manual installation.
+
```

Comment:
suggest another comma after `clients`.
"We have found that the efficiency gains of spending less time in server administration translates to more value to our clients, empowering additional effort for the strategy, design, and development for which they hired us."